fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -78,4 +78,14 @@ object FeatureFlags {
      * Enable Meta attribution.
      */
     val metaAttributionEnabled = Config.channel.isNightlyOrDebug
+
+    /**
+     * Enable Toolbar Redesign components and behaviors ready for Nightly.
+     */
+    val completeToolbarRedesignEnabled = Config.channel.isNightlyOrDebug
+
+    /**
+     * Enable Toolbar Redesign partial components and behaviors.
+     */
+    const val incompleteToolbarRedesignEnabled = false
```

Comment:
shouldn't we also enable it for dev build? I don't think it's going to work in combination with `var enableIncompleteToolbarRedesign` well. Though I might be mistaken

Reply:
good catch! switched to `channel.isDebug` as we want devs to turn it off and on.